addons/skin.estuary/xml/Variables.xml

```diff
@@ -160,7 +160,8 @@
 		<value>$LOCALIZE[16018]</value>
 	</variable>
 	<variable name="AddonsListIconVar">
-		<value condition="!String.IsEmpty(ListItem.AddonBroken)">icons/addonstatus/disable.png</value>
+		<value condition="(String.IsEqual(ListItem.AddonLifecycleType,deprecated) | String.IsEqual(ListItem.AddonLifecycleType,broken))+ !ListItem.Property(addon.isenabled)">icons/addonstatus/disable.png</value>
+		<value condition="ListItem.Property(addon.isenabled)">$INFO[ListItem.AddonLifecycleType,icons/addonstatus/enabled-,.png]</value>
```

Comment:
sorry, this is not going to works as $INFO[ListItem.AddonLifecycleType] needs to return a localized string

Reply:
Thought to reduce the work amount and where I asked you on slack to use this identifier to get related image about.
Also not seen them as usable text, only as type identifiers.

Can then use as;
```xml
<value condition="ListItem.Property(addon.isenabled)" + String.IsEqual(ListItem.AddonLifecycleType,$LOCALIZE[xxxx1])">icons/addonstatus/enabled-normal.png</value>
<value condition="ListItem.Property(addon.isenabled)" + String.IsEqual(ListItem.AddonLifecycleType,$LOCALIZE[xxxx2])">icons/addonstatus/enabled-deprecated.png</value>
<value condition="ListItem.Property(addon.isenabled)" + String.IsEqual(ListItem.AddonLifecycleType,$LOCALIZE[xxxx3])">icons/addonstatus/enabled-broken.png</value>
```
Can be this much string compares then everywhere not touch the performance?

-----------------------------

Thought before for use e.g. like this
Also thought for this cases that addon can use his own labels, e.g.
```xml
<control type="label">
  <left>85</left>
  <right>40</right>
  <bottom>0</bottom>
  <aligny>center</aligny>
  <visible>String.IsEqual(ListItem.AddonLifecycleType,normal)</visible>
  <label>31001</label>
</control>
<control type="label">
  <left>85</left>
  <right>40</right>
  <bottom>0</bottom>
  <aligny>center</aligny>
  <visible>String.IsEqual(ListItem.AddonLifecycleType,deprecated)</visible>
  <label>31002</label>
</control>
<control type="label">
  <left>85</left>
  <right>40</right>
  <bottom>0</bottom>
  <aligny>center</aligny>
  <visible>String.IsEqual(ListItem.AddonLifecycleType,broken)</visible>
  <label>31003</label>
</control>
```

Reply:
instead of providing ListItem.AddonLifecycleType as an infolabel and then using String.IsEqual() to convert it to a boolean,
why not provide an infobool directly?
something like `ListItem.HasAddonLifecycleType(deprecated)`